settings.ini

```diff
@@ -6,7 +6,7 @@ min_python = 3.8
 license = apache2
 black_formatting = False
 requirements = fastcore>=1.5.33 anthropic toolslm
-dev_requirements = google-auth
+dev_requirements = google-auth boto3
```

Comment:
What's boto3 added for?

Reply:
The AWS examples at the bottom need it.

Reply:
Could we replace those with something that doesn't bring in an additional dev dep please?

Reply:
Without boto3 Claude can't use AWS:
![image](https://github.com/user-attachments/assets/5ee40386-2437-4c7f-a83e-779b2d118234)

https://github.com/anthropics/anthropic-sdk-python/blob/main/src/anthropic/lib/bedrock/_auth.py#L21

They don't include it as a dependency, but if you want the AWS example to work, we need boto3.

Reply:
I'm ok with just ignoring this and not including the dependency either.

Reply:
Thank you -- sorry I was a bit slow, I understand the issue now!